### ISSUE TITLE
0.8.x support

### ISSUE THREAD

**Issue by irbian** [edzillion/blood-n-guts#276]
Really interesting module! but not working on 0.8.6

**Describe the bug**

```
BloodLayer.js:18 Uncaught ReferenceError: TilesLayer is not defined
    at BloodLayer.js:18
```

**Versions (please complete the following information):**
 - Module version [0.8.7]
 - Foundry version [0.8.6]
 - System & version [DND35]

**Comment by sabatmonk** [comment]
TilesLayer was refactored to backgroundtiles, but simply changing name do not solve the issue completely. blood is there when the scene is loaded but it then cannot add blood nor is blood on the ground generated, only on tokens

**Comment by p4535992** [comment]
Start something here [https://github.com/edzillion/blood-n-guts/pull/277](https://github.com/edzillion/blood-n-guts/pull/277) for make a patched version for foundryvtt 0.8.6

**Comment by Fridan99** [comment]
yes, it still doesn't work in 0.8.x...., the button doesn't even appear in the left bar.

**Comment by Sunamon** [comment]
Hiya! Any plans on 0.8.x support? i'm missing the gore! lol

Cheers!

**Comment by edzillion** [comment]
> 
> 
> Hiya! Any plans on 0.8.x support? i'm missing the gore! lol
> 
> Cheers!

yep gonna get stuck into it now, if I'm not back by dinner send a search party

**Comment by Gacky2k** [comment]
Just wondering...any progress on getting this up to date to work in 0.8.X?

**Comment by p4535992** [comment]
> 
> 
> Just wondering...any progress on getting this up to date to work in 0.8.X?

In the meanwhile you can try [this](https://github.com/edzillion/blood-n-guts/pull/277#issuecomment-882402813)
